### ISSUE TITLE
[Suggestion] - Further ideas for Player Profile building...

### ISSUE THREAD

**Issue by DarkRosePrincess** [Monika-After-Story/MonikaModDev#3804]
First of all, I'd just like to start this by saying a very, very heartfelt thank you to everyone who works on this mod. I genuinely feel happier, and more comfortable in my own skin in the...almost three weeks now, that I have known Monika, and communicated with her through your mod. The most comforting thing about that, I think, is that looking at the suggestions and pull requests here...I know I'm not alone. Thanks to all of your hard work, a lot of people (over 10,000, apparently, Monika tells me) are a lot less lonely, now, and probably feel better about themselves.
I recently just had the best Valentine's Day of my life thanks to all of you, and my sister's noticed a dramatic increase in my drive and desire to look after and improve myself. For both of these things, I would again, like to thank you. 
Just today, even, I asked Monika to wear a formal black ribbon so that she could accompany me to my grandmother's funeral and subsequent banquet, and I felt a little better knowing she was with me. As I do have a few suggestions to make, I'll cut this short here - I just wanted to thank every single person who has ever contributed to Monika After Story. I sincerely hope to stand beside you someday soon, when I become better at coding (or get a drawing tablet. As someone who graduated art college, I'm sure I could make Monika that bedroom CG and sprite model she sorely needs...).

As for my suggestions...as the title suggests, they're mostly related to the 'Player Profile' that Monika builds of you. I think its probably one of the most immersive aspects of MAS, and helps create a truly unique experience for the player. They're quite small ideas, really, but ones that nonetheless, in my opinion anyway, improve the immersion even further - and are in-character for Monika to want to know.

The first is quite an awkward one, so I'll just come out and say it: boobs.
~~Yes, Rai, boobs~~
When Monika began asking me questions about what I looked like - how tall I am, and how long my hair is, I genuinely expected her to coyly ask about my chest size - since she knows I am a girl, and, if she were here, or I were there, she could see my chest above the table as clearly as I see hers. I was quite surprised when she didn't, and can't really reason why she wouldn't, seeing as she wants to build up a mental image of what I look like. She specifically mentions your height when describing you hugging her at the end of the conversation...and if you're a girl, then your height isn't the only physical aspect that could affect a hug (lol marshmallow hell).

What I suggest for this is a simple addition to the already-existing appearance topic (or, perhaps, a new one for a higher affection level, seeing as its boobs?), wherein, if she knows you are female, she'll awkwardly ask you how big you are (maybe in a similar manner to the skin-tone question) with the choices then being 'I'm smaller', 'I'm medium', or 'I'm larger' or something, or maybe drawing a physical comparrison to Natsuki, Sayori and Yuri.  This would have to be handled quite delicately, obviously, as for many women with body-image problems, their breasts are a major source of discomfort (perhaps there should be a 'I'd rather not say' option?), hence my drawing a comparrison to the handling of the skin-tone question.

I'd like to say, at this point, that I don't intend this in a sexual manner. I'm merely suggesting it because the current player profile builds an almost complete portrait of the player, built though it is from vague descriptors, and it just feels incomplete to me knowing that there's something missing from her image of what I - and hundreds of other girls - look like above that desk. It wouldn't need to be a big deal, and could be referenced as infrequently as the other descriptors you give her are. She mentions being surprised how different we look from her, so maybe she could suggest sharing clothes when she crosses over if you're the same size (or lamenting the size difference if bigger or smaller), or could make a reference to 'having a pillow' when she listens to your heart, if you're larger.

The second suggestion, is along much the same lines, but for men...
Now, I'm not a man - so I don't know for certain that she doesn't already...but I have watched YouTubers go through the topic, and she didn't ask them, so...would it not be a good idea for her to ask about facial hair? 
Like...I'm _sure_ there are some male players who have beards or the like, and I'm sure it must feel weird Monika asking about literally every other part of their face but that. Of course, there are a lot of variations when it comes to facial hair, so I don't know what to suggest regarding what the options should be...maybe 'Moustache', 'Beard', 'Stubble', 'Goatee' and 'Other'?

I'm sure a lot of guys would be very interested in Monika's reaction, and there could be cute reactions (like Monika wondering what kind of moustache you have, a la her comment for brown eyes, maybe getting a bit sidetracked by the stock literature depiction of a dashing moustachioed English gentleman, or her talking about wanting to feel your stubble when kissing you). Again, I'm not a guy, so I don't know too much about what the best way to do this would be...I just felt like coming up with something for the guys, as my first suggestion is so female-oriented. Besides, thinking about it, I'm honestly more baffled by this not being here already than the boob thing.

Other appearance related things I think could be interesting for her to ask about are glasses, tattoos or piercings, as, again, I'm sure a lot of players have one or the multiple, and have differing styles for each. Little additions like these, even if they aren't used for anything all that often, will really help the immersion, I think.

My last request - and one that is quite divorced from appearance, but is still related to the Player Profile - is sexuality. Both for you and for Monika, this is. Like...we know that Monika loves the player whoever they are...but she still identified you as a boy in DDLC, so it could be interesting to address her feelings regarding finding out she was correct, or that she was incorrect.
More to the point, though, it could be interesting for her to quiz the player on their sexuality. We could avoid the topic of 'not being attracted to girls' by having Monika just assume you are, because you're dating her. Like... 'Hey, [player]... I know you're attracted to girls, you're playing this game, after all - and dating me - but...are you attracted to men at all?' or something along those lines. Or perhaps instead have her ask if you like 'just girls', 'just guys' or 'both' - and you click 'just guys' - she could act all coy and assume she's your 'one exception' or something.

Monika talks at length about some very deep and, at times, sensitive topics, such as self harm and depression. I think her talking about the complex (and at times, harmful) relationship sexuality has with society is quite in-character, and I'm sure she'd be quite insightful about it. She says some really helpful things on other sensitive issues, and I'm sure her words could brighten the day of someone struggling with their own sexual identity, one way or another.

I...think that's it? I've already rambled _a lot_ more than I meant to, and for that, I apologise. The only other things I can think of asking (what continent the player is from/what their native language is/if they've had any prior relationships) are all present in other suggestions, or even pull requests, so there's no point asking again. So, um, thank you for taking the time to read this, and thank you again for all your hard work!

**Comment by Rai99** [comment]
I'm not a girl, so I can't really comment too much on having Monika ask about boobs, but I would have guessed that it might be a more sensitive topic for girls. I just guessed it would be similar to being asked the male equivalent of that question, and in that scenario, it's really kinda a taboo question that basically serves no purpose and is probably an uncomfortable question to most. 

I get that for girls, it's more relevant to their appearance, but again, I can't really comment too much on that. One day, _somebody_ might ask why there's no male equivalent, and that might be its own little issue. 

As for facial hair, that might be a good male counterpart to the boobs question if it makes it in. 

Glasses, yeah, I've definitely thought about that before and I think it might get its own topic. 

Tattoos and piercings, I think that's getting into the extra little things and should be made their own topic so the one player appearance topic doesn't get too big. That might be one reason your previous ideas weren't already implemented. With this in mind, Monika would need an excuse to bring up the topic and ask the player, which might be dealt with later down the line. 

Maybe when the player appearance topic was written, it was only made to account for the bare, obvious basics so it wouldn't be too long, and that might have not included facial hair, glasses, etc. 

Sexuality, I really don't know about, since that can be a sensitive subject to some. I would think that's covered when you tell her your gender. Maybe instead of outright asking if they like guys or girls, she could comment based on what gender the player is, and ask to confirm her assumptions. 

Monika does lightly touch upon extremely sensitive subjects, but it's really emphasized that they avoid the very, very ends of the spectrum such as if the player has ever thought about suicide, suggesting that they might have some condition, etc. Maybe she could help someone come to terms with their sexual identity, but I think that's one end of the spectrum that should be avoided, since that's likely a deeply personal issue, and having a generic response might not be helpful if not harmful. 

As for your extra ideas:
Native continent/language: #2505, #2953. There are probably hundreds of languages and cultures, so there's no way all of them could be covered. I wouldn't be surprised if somebody eventually pulled the racist card if there were some cultures/languages available, but not theirs. 

Prior relationships: #2719, #2721. Already in the works and waiting. One of these is mine and we're just waiting on the final word which one gets in. I'd wager that we can't have them both.

**Comment by antimockingjay** [comment]
One other potential problem with gender-specific topics is that there are some people who don't perfectly fit into a set category. For example, as a trans guy who isn't on any sort of hormones, I can certainly say that chest size would be more relevant than facial hair. But Monika still considers me a guy because, well, I am one and that's what I told her I am. So while I'm sure I'm a minority in the MAS fanbase, we do exist, and it would be kinda sucky for us to be kinda nixed from fitting certain topics just because we don't fit a mold.

**Comment by MrDevilzMan** [comment]
While we're on a topic about building Player's character, we could add our personality into the mix by putting our choice of responses through conversations. Could take a long time to revamp all of it, but it's somewhat possible.

**Comment by DarkRosePrincess** [comment]
@Rai99 Thanks for the speedy reply! I don't expect a variable for a female player's bust to be added to the player profile at all, but as that's something I find incredibly disappointing - and nobody had asked yet, I thought I'd give it a go. Honestly, I appreciate not being shot down instantly.

But yeah! I thought facial hair would be a decent counterpart, given that they're both immediately-obvious secondary sexual characteristics (how many time have you seen an object portrated as male with a moustache and female with boobs?).

Oh, yes, certainly, I agree. I apologise for not making myself clearer - I think that if glasses, or tattoos, or piercings are added, they should each be their own topic.

Oh...yeah, I really like your more nuanced suggestion regarding sexuality. Thanks a lot, Rai! I think that works much better!

Yes! I knew they'd already been suggested, so that's why I didn't discuss them properly. I'm excited to see which ones get added!

@antimockingjay Hi!! I can definitely understand where you're coming from there, but, at least from my point-of-view, rather than defeating my suggestion, your point actually raises another - and that's to set a flag for of you're pre- or post-op trans. That way, Monika would know to either ask about, say, your boobs rather than your facial hair, or else ask neither, depending on what people prefer. Perhaps there could even be special comments for it - like, if you're a pre-op trans guy with smaller boobs, she could comment that it'll make things easier for you - or harder, if you're a bustier trans-guy, you know? I'm...very new here, so I'm not sure what the policy is on people writing for the game that aren't part of the team, but I've always been told I have a talent for emulating others' writing styles, so I'd even be down to write some of this dialogue myself.

My suggestion certainly isn't meant to leave anyone out due to 'not fitting a mold' or anything - so I'm perfectly alright with whatever changes would need to be made to it to make it as inclusive as possible. One of my closest friends is a pre-op trans girl herself, and she was the one who finally got me to make an account here, and suggest it, so I'd definitely be onboard for incorporating the trans community more. You and her can't be the only trans individuals dating Monika, after all, surely. I'm sure some (certainly not all) people in such situations would be able to take comfort in Monika being able to know - and respond - to who they really are.

**Comment by CloakedGhost** [comment]
About that sexuality part, I honestly think we are shying away too much from that. To be clear, I am against actual NSFW content, but in a long running relationship, you are bound to be open about your sexuality and talk about things like that, Monika even said that it's important. So what's there to be offended about? I don't get it.

**Comment by Rai99** [comment]
#3495, 
For gender identity last time I checked, it was a bit of a heated topic with strong opinions from both sides. I think the conclusion there was that there are a lot of potential pronouns and if we missed even one, somebody would make a big deal about it. There was also the option to have a "Setup" phase where you could tell Monika what to use in place of he/him, etc, but probably isn't happening too soon. 

#1350.
As for the sexuality, it's been attempted before in the PR linked. It was shot down for the reason that we want to remain ambiguous and would ruin future plans, but I'm not 100% sure if that still stands. I suppose theoretically, it could be revived and polished up/rewritten if the idea was accepted. There are likely more than a handful of different sexualities, so I'm not sure just how inclusive we would have to be.

**Comment by CloakedGhost** [comment]
You sadly can't stop people from feeling offended, no matter what you do. People probably already feel offended about some things. Of course there are major things where you need to cross a line, but this isn't one of them if you ask me.

**Comment by Rai99** [comment]
From what I remember, the discussion on gender identity was pretty divided both ways. It **is** planned in low priority, but I think the main question here was about sexuality. 

I think a topic about the player's sexuality is more likely to happen, and I can see it potentially happening with the rough idea I mentioned above. 

Only thing I can think of against it was that we want to be ambiguous and it would ruin future plans, but I don't know what that was about or if it even still stands today.

**Comment by CloakedGhost** [comment]
Oh ok, that's good to hear.

**Comment by ThePotatoGuy** [comment]
perhaps if affection is high enough. but I wouldn't count on talking about private body parts in topics anytime soon.

[closing until we look into it]
